docs/core/distribution-packaging.md

```diff
@@ -16,7 +16,7 @@ As .NET 5 (and .NET Core) and later versions become available on more and more p
 When installed, .NET consists of several components that are laid out as follows in the file system:
 
 ```
-{dotnet_root}                                     (*)
+{dotnet_root}                    (0)
```

Comment:
I think the asterias should be maintained as the `dotnet_root` is used by multiple packages.

```suggestion
{dotnet_root}                    (0)               (*)
```